index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
I'm really unclear about where it is supposed to be placed. The documentation is very lacking in this area (and in general, really).